src/composition.rs

```diff
@@ -633,15 +639,11 @@ impl Composition {
         }
     }
 
-    pub(crate) fn add_measurement(
+    pub(crate) fn set_measurement(
         &mut self,
         measurement: String,
     ) -> CompositionResult {
-        if self.analyzer.measurement.is_some() {
-            return Err(());
-        } else {
-            self.analyzer.measurement = Some(measurement)
-        }
+        self.analyzer.measurement = Some(measurement);
         self.sync();
         Ok(())
     }
```

Comment:
Do we expect this method to error ever? If not should we change its signature to not return an error?

Reply:
All of the methods on the struct Composition, which then modify the composition as a result of an `executeCommand`, will return a `CompositionResult`.

In this particular case only, we don't have a scenario which returns an Err.